cli/commands/project_sync.go

```diff
@@ -123,19 +141,39 @@ func (cmd *ProjectSync) RunStart(ctx *cli.Context) error {
 			unisonArgs = append(unisonArgs, "-ignore", ignore)
 		}
 	}
-
 	cmd.out.Verbose.Printf("Unison Args: %s", strings.Join(unisonArgs[:], " "))
 	if err = exec.Command("unison", unisonArgs...).Start(); err != nil {
 		cmd.out.Error.Fatalf("Error starting local unison process: %v", err)
 	}
-
 	cmd.WaitForSyncInit(logFile, ctx.Int("initial-sync-timeout"), ctx.Int("initial-sync-wait"))
+}
 
-	return nil
+// For systems that have native container/volume support
+func (cmd *ProjectSync) SetupBindVolume(volumeName string) {
+
+	cmd.out.Info.Printf("Starting local bind volume: %s", volumeName)
+	exec.Command("docker", "volume", "rm", volumeName).Run()
+
+	if workingDir, err := os.Getwd(); err == nil {
+		volumeArgs := []string{
+			"volume", "create",
+			"--opt", "type=none",
+			"--opt", fmt.Sprintf("device=%s", workingDir),
+			"--opt", "o=bind",
+			volumeName,
+		}
+		exec.Command("docker", volumeArgs...).Run()
+	} else {
+		cmd.out.Error.Fatalf("Error resolving the working directory for volume creation: %v", err)
+	}
 }
 
-// Start the unison sync process
 func (cmd *ProjectSync) RunStop(ctx *cli.Context) error {
+	if runtime.GOOS == "linux" {
+		cmd.out.Info.Println("No unison container to stop, using local bind volume")
```

Comment:
Since we are creating the volume as an external, shouldn't this stop the volume for them?

And if so, as long as any action is taken I think logging about no-op should be verbose.

Reply:
Volumes are not start or stopped, they just exist. Are you suggesting we do the volume remove here in the stop?

Reply:
Nope, you are right, we should probably not do a volume remove here.